pkgs/tools/backup/tsm-client/default.nix

```diff
@@ -76,22 +82,23 @@ let
 
   unwrapped = stdenv.mkDerivation rec {
     name = "tsm-client-${version}-unwrapped";
-    version = "8.1.8.0";
+    version = "8.1.12.0";
     src = fetchurl {
-      url = "ftp://public.dhe.ibm.com/storage/tivoli-storage-management/maintenance/client/v8r1/Linux/LinuxX86_DEB/BA/v818/${version}-TIV-TSMBAC-LinuxX86_DEB.tar";
-      sha256 = "0c1d0jm0i7qjd314nhj2vj8fs7sncm1x2n4d6dg4049jniyvjhpk";
+      url = "https://public.dhe.ibm.com/storage/tivoli-storage-management/maintenance/client/v8r1/Linux/LinuxX86_DEB/BA/v8112/${version}-TIV-TSMBAC-LinuxX86_DEB.tar";
```

Comment:
Please subsitute the version in the URL. lib.replaceString should be handy here.

Reply:
I failed with `replaceString`, but `splitString` is able to dissect the version number.